pkg/operatorsource/operatorsource.go

```diff
@@ -0,0 +1,16 @@
+package operatorsource
+
+// DatastoreLabel is the label used in a CatalogSourceConfig to indicate that
+// the resulting CatalogSource acts as the datastore for the OperatorSource
+// if it is set to "true".
+const DatastoreLabel string = "opsrc-datastore"
+
+// OpsrcOwnerNameLabel is the label used to mark ownership over resources
+// that are owned by the OperatorSource. When this label is set, the reconciler
+// should handle these resources when the OperatorSource is deleted.
+const OpsrcOwnerNameLabel string = "opsrc-owner-name"
+
+// OpsrcOwnerNamespaceLabel is the label used to mark ownership over resources
+// that are owned by the OperatorSource. When this label is set, the reconciler
+// should handle these resources when the OperatorSource is deleted.
+const OpsrcOwnerNamespaceLabel string = "opsrc-owner-namespace"
```

Comment:
Are `OpsrcOwnerNameLabel` and `OpsrcOwnerNamespaceLabel` still being used anywhere?

Reply:
They are only used in the unit/e2e tests now actually. What was their original purpose?

Reply:
Same purpose as the other labels. They acted as `OwnerReferences` for objects owned by `OperatorSources`.

Reply:
That should be happening given how you have abstracted it but it will be using the new label. I think you might have to put different labels depending on if it is a `CatalogSourceConfig` or `OperatorSource` otherwise deletion of the resources won't happen. Unfortunate these labels (except for CatalogSources created in  namespaces other than the watched one) will be going away once we go back to using `OwnerReferences`. Will it make it easier for you if open a PR that does that?